react/Image.tsx

```diff
@@ -18,6 +19,13 @@ export interface ImageProps extends ImgHTMLAttributes<HTMLImageElement> {
   blockClass?: string
   experimentalPreventLayoutShift?: boolean
   link?: Link
+  width?: string
+  src: string
+  alt?: string
+  height?: string
+  srcSet?: string
+  sizes?: string
+  title?: string
```

Comment:
Adding this definitions here are redundant, since this interface is extending `ImgHTMLAttributes<HTMLImageElement>`. This big type is a built-in one, so it's better to use it instead of adding each property individually.

We just need to add the ones that are not already part of the specifications of an `<a>` tag, such as `maxWidth`, `blockClass`, etc.